snuba/cli/multistorage_consumer.py

```diff
@@ -174,7 +190,14 @@ def multistorage_consumer(
         # is no way to specify otherwise, at writing) and b.) all storages are
         # located on the same Kafka cluster (validated above.)
         producer = ConfluentKafkaProducer(
-            build_kafka_producer_configuration(storage_keys[0])
+            build_kafka_producer_configuration(
+                storage_keys[0],
+                storages[storage_keys[0]]
+                .get_table_writer()
+                .get_stream_loader()
+                .get_default_topic_spec()
+                .topic,
+            )
```

Comment:
Having a utility function that takes a storage and returns the topic id may be useful to avoid all this boilerplate

Reply:
I'm actually dubious this line is correct at all. Since it's the producer for the commit log shouldn't it be `get_commit_log_topic_spec` not `get_default_topic_spec`? Still, I left it as default because I don't want to introduce logical changes in the PR and that was what was getting applied earlier.

Reply:
You are right,  before the broker was associated to the storage (so default and commit log produced the same broker so it was correct). Now this is going to be associated with the logical topic. So you should change it to the get_commit_log_topic_spec. I would make the change.

Reply:
Updated. Also changed subscriptions and replacements to use the right topic configuration.